components/post-preview.js

```diff
@@ -0,0 +1,23 @@
+export default function PostPreview({
+  title,
+  excerpt,
+  tags,
+}) {
+  return (
+    <div className="p-6 flex flex-col items-start">
+      {tags.map((tag) => (
+        <span className="inline-block py-1 px-2 rounded bg-red-50 text-red-500 text-xs font-medium tracking-widest">{tag}</span>
```

Comment:
Looks like the parent is declaring that it's flex, so you'll want to try `inline-flex` instead of `inline-block`